### ISSUE TITLE
Firing a weapon straight up/down causes projectile spread to land in a straight line

### ISSUE THREAD

**Issue by Silverfang22** [JKGDevs/JediKnightGalaxies#192]
The further straight up and down you aim the less horizontal spread your shots will have.

http://i.imgur.com/0BdDmlT.jpg

**Comment by DarthFutuza** [comment]
Is this necessarily a bad thing?  I mean I know it wasn't initially intended, but hitting things straight above you is already hard enough.

**Comment by Silverfang22** [comment]
Yes this is a bad thing, it looks strange and really only makes a difference with shotguns (where you may want the spread to hit airborne targets anyway)
